app/views/taxon_changes/edit.html.haml

```diff
@@ -0,0 +1,13 @@
+- content_for( :title ) do
+  - @title = capture do
+    =t :editing
+    =t @taxon_change.class.name.underscore
+    = surround '(', ')' do
+      = @taxon_change.committed? ? t(:committed_on, date: l( @taxon_change.committed_on ) ) : t(:draft)
+  = strip_tags @title
+= render partial: "form_head"
+.last.column.span-24#pageheader
+  %span.breadcrumbs
+    = link_to t( :back_to_x, :noun => t( @taxon_change.class.name.underscore ) ), taxon_change_path( @taxon_change ), class: 'back crumb'
```

Comment:
I'd just use `t( "back" )` or add `t( "back_to_taxon_change" ) or something. This is the kind of thing that drives translators crazy with gender agreement and such.